src/main/generated/pack.mcmeta

```diff
@@ -3,6 +3,6 @@
     "description": {
       "translate": "pack.forge.description"
     },
-    "pack_format": 34
+    "pack_format": 48
```

Comment:
Since this seems to be a generated file, make sure it is accounted for in the generator. Right now, it seems to be manually edited.